src/box/lua/net_box.c

```diff
@@ -1864,7 +1864,7 @@ netbox_perform_request(struct lua_State *L)
 			netbox_request_unregister(&request);
 			netbox_request_destroy(&request);
 			luaL_testcancel(L);
-			diag_set(ClientError, ER_TIMEOUT);
+			diag_set(TimedOut);
```

Comment:
There are other functions that throw `ER_TIMEOUT` error - `luaT_netbox_request_wait_result` and `luaT_netbox_request_iterator_next` - they are used by asynchronous requests (`{is_async = true}`). Could you please fix them as well in the scope of this patch?

Reply:
Fixed